spring-cloud-gcp-core/src/main/java/org/springframework/cloud/gcp/core/UnexistingProjectIdException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright 2017 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.core;
+
+import org.springframework.core.NestedRuntimeException;
+
+/**
+ * Signals the lack of a configured GCP project ID.
+ *
+ * @author João André Martins
+ */
+public class UnexistingProjectIdException extends NestedRuntimeException {
```

Comment:
`NoProjectIdException` was actually my first choice, but I thought it was too negative :sweat_smile: 

I like `MissingProjectIdException`